modelon/impact/client/experiment_definition.py

```diff
@@ -25,6 +25,16 @@ def _case_to_identifier_dict(case):
     }
 
 
+def _single_true(iterable):
+    i = iter(iterable)
+    return any(i) and not any(i)
+
+
+def _assert_unique_exp_initialization(*args):
+    if not _single_true(args):
+        raise ValueError("An experiment can only be initialized from one entity!")
```

Comment:
Maybe this error reporting could be made a little more precise. For example you could like to see all entities you are initializing from to understand what is wrong. I am thinking something like:
```
def _assert_unique_exp_initialization(*initializing_from):
  initializing_from = [entity for entity in initializing_from if entity is not None]
  if len(initializing_from) > 1:
     raise ValueError(f"An experiment can only be initialized from one entity. Experiment is configured to initialize from {initializing_from}")
```